lib/drivers/bayerContourNext.js

```diff
@@ -44,111 +45,117 @@ module.exports = function (config) {
    * TODO This info should be collected in the initial phase cause we can't assert if
    *      it is the same for all contour devices
    * */
-  var REPORT_BYTES = {
-    reportID: 0x41, // A
-    //checksum: 0x00, // Checksum is not been used by bayer contour device
-    hostID  : 0x42, // B
-    deviceID: 0x43  // C
-  };
+  var MAGIC_HEADER = 'ABC';
 
   /* end */
 
   var ASCII_CONTROL = {
     ACK : 0x06,
-    NAK : 0x15,
+    CR : 0x0D,
     ENQ : 0x05,
-    STX : 0x02,
+    EOT : 0x04,
     ETB : 0x17,
     ETX : 0x03,
-    EOT : 0x04,
-    TER : 0x4c
+    LF : 0x0A,
+    NAK : 0x15,
+    STX : 0x02
   };
 
+  var astmFrameBuffer = {
+    reset: function(){
+      this.bytes = new Uint8Array(0);
+      this.valid = false;
+      this.frame_len = 0;
+      this.payload = null;
+      return this;
```

Comment:
Is it necessary to return here, as we don't seem to be using the returned value?